src/CXXTests/testSidreStorage.cc

```diff
@@ -0,0 +1,645 @@
+
+  //myData.printDataStore();
+
+  //for (int i = 0; i < n; i++)
+    EXPECT_EQ(testField[0].xx(), rawSidreData[0]);
+}
+
+TEST(SidreDataCollectionTestDim1ThirdRankTensor, ThirdRankTensor)
+{
+  Spheral::SidreDataCollection myData;
+  int n = 10;
+
+  Spheral::NodeList<Spheral::Dim<1>> makeNodeList("test bed", n, 0);
+  Spheral::Field<Spheral::Dim<1>, Spheral::Dim<1>::ThirdRankTensor> testField("test field", makeNodeList);
+  for (int i = 0; i < n; i++)
+    testField[i] = Spheral::Dim<1>::ThirdRankTensor(i);
+
+  //std::cout << testField[0](0, 0, 0) << std::endl;
+  
+  double* rawSidreData = myData.alloc_view("SidreTest", testField)->getData();
+
+  //myData.printDataStore();
+
+  //for (int i = 0; i < n; i++)
+    EXPECT_EQ(testField[0](0,0,0), rawSidreData[0]);
+}
+
+TEST(SidreDataCollectionTestDim1FourthRankTensor, FourthRankTensor)
+{
+  Spheral::SidreDataCollection myData;
+  int n = 10;
+
+  Spheral::NodeList<Spheral::Dim<1>> makeNodeList("test bed", n, 0);
+  Spheral::Field<Spheral::Dim<1>, Spheral::Dim<1>::FourthRankTensor> testField("test field", makeNodeList);
+  for (int i = 0; i < n; i++)
+    testField[i] = Spheral::Dim<1>::FourthRankTensor(i);
+
+  //std::cout << testField[0](0, 0, 0, 0) << std::endl;
+  
+  double* rawSidreData = myData.alloc_view("SidreTest", testField)->getData();
+
+  //myData.printDataStore();
+
+  //for (int i = 0; i < n; i++)
+    EXPECT_EQ(testField[0](0, 0, 0, 0), rawSidreData[0]);
+}
+
+TEST(SidreDataCollectionTestDim1FifthRankTensor, FifthRankTensor)
+{
+  Spheral::SidreDataCollection myData;
+  int n = 10;
+
+  Spheral::NodeList<Spheral::Dim<1>> makeNodeList("test bed", n, 0);
+  Spheral::Field<Spheral::Dim<1>, Spheral::Dim<1>::FifthRankTensor> testField("test field", makeNodeList);
+  for (int i = 0; i < n; i++)
+    testField[i] = Spheral::Dim<1>::FifthRankTensor(i);
+
+  //std::cout << testField[0](0, 0, 0, 0, 0) << std::endl;
+  
+  double* rawSidreData = myData.alloc_view("SidreTest", testField)->getData();
+
+  //myData.printDataStore();
+
+  //for (int i = 0; i < n; i++)
+    EXPECT_EQ(testField[0](0, 0, 0, 0, 0), rawSidreData[0]);
+}
+
+// TEST(SidreDataCollectionTestDim1FacetedVolume, FacetedVolume)
+// {
+//   Spheral::SidreDataCollection myData;
+//   int n = 10;
+//   std::vector<Spheral::Dim<1>::Vector> temp;
+
+//   Spheral::NodeList<Spheral::Dim<1>> makeNodeList("test bed", n, 0);
+//   Spheral::Field<Spheral::Dim<1>, Spheral::Dim<1>::FacetedVolume> testField("test field", makeNodeList);
+//   for (int i = 0; i < n; i++)
+//     temp.push_back(Spheral::Dim<1>::Vector(i));
+
+//   for (int i = 0; i < n; i++)
+//     std::cout << temp[i] << " ";
+//   std::cout << std::endl;
+
+//   testField[0] = Spheral::Dim<1>::FacetedVolume(temp);
+    
+  
+//   double* rawSidreData = myData.alloc_view("SidreTest", testField)->getData();
+
+//   myData.printDataStore();
+
+//   // for (int i = 0; i < n; i++)
+//   //   EXPECT_EQ(testField[0](0, 0, 0, 0, 0), rawSidreData[0]);
+// }
```

Comment:
Please make a pass through all of the touched files to remove any old / unused or commented out code like this.